app/models/saved_claim/burial.rb

```diff
@@ -1,11 +1,29 @@
 # frozen_string_literal: true
 class SavedClaim::Burial < SavedClaim
   FORM = '21P-530'
+  CONFIRMATION = 'PEN'
+  PERSISTENT_CLASS = PersistentAttachment::PensionBurial
+  ATTACHMENT_KEYS = [:transportationReceipts, :deathCertificate].freeze
+
   def regional_office
     PensionBurial::ProcessingOffice.address_for(open_struct_form.claimantAddress.postalCode)
   end
 
-  def confirmation_number
-    "V-BUR-#{id}#{guid[0..6]}".upcase
+  def to_pdf
+    #     # return FormToPDF::Blahblah.convert(open_struct_form)
```

Comment:
Obviously commented out so this may be ignorable :) but would this be extracted into a class? maybe part of the same class/lib that the convert to pdf task uses? Only concern is relying on system calls and reliably catching errors from them.